fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/cache/CommonTokenValuesCacheImpl.java

```diff
@@ -198,6 +215,53 @@ public void resolveTokenValues(Collection<ResourceTokenValueRec> tokenValues,
         }
     }
 
+    @Override
+    public void resolveCanonicalValues(Collection<ResourceProfileRec> profileValues,
+        List<ResourceProfileRec> misses) {
+        // Make one pass over the collection and resolve as much as we can in one go. Anything
+        // we can't resolve gets put into the corresponding missing lists. Worst case is two passes, when
+        // there's nothing in the local cache and we have to then look up everything in the shared cache
+
+        // See what we have currently in our thread-local cache
+        LinkedHashMap<String,Integer> canMap = canonicalValues.get();
+
+        List<String> foundKeys = new ArrayList<>(profileValues.size()); // for updating LRU
+        List<ResourceProfileRec> needToFind = new ArrayList<>(profileValues.size()); // for the canonical values we haven't yet found
+        for (ResourceProfileRec tv: profileValues) {
```

Comment:
i usually write these with a space between the var name and the `:` ...probably not important but thought i'd mention in case we want consistent style

Reply:
Never done that. For me that's like putting a space before a ;

Reply:
FWIW, with the space is what i've always seen others do too: 
 https://docs.oracle.com/javase/8/docs/technotes/guides/language/foreach.html